src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -991,12 +993,12 @@ void goto_analyzer_parse_optionst::help()
     " --dependence-graph-vs        dependencies between instructions using VSD\n" // NOLINT(*)
     "\n"
     "Variable sensitivity domain (VSD) options:\n"
-    " --vsd-structs                struct field sensitive analysis\n"
-    " --vsd-arrays                 array entry sensitive analysis\n"
-    " --vsd-pointers               pointer sensitive analysis\n"
-    " --vsd-value-sets             use value sets\n"
+    " --vsd-values                 value tracking - constants|intervals|set-of-constants\n"
+    " --vsd-structs                struct field sensitive analysis - top-bottom|every-field\n"
+    " --vsd-arrays                 array entry sensitive analysis - top-bottom|every-element\n"
+    " --vsd-pointers               pointer sensitive analysis - top-bottom|constants|value-set\n"
+    " --vsd-unions                 union sensitive analysis - top-bottom\n"
```

Comment:
Had discussed this with @martin-cs recently: there is a fair abundance of options, and to noobs like myself it's not obvious which ones to choose. As a first step, it would actually help to indicate the default choices in the help output. Could you please add that information?

Reply:
I have a goal of writing a PR that improves the documentation in the manual (to explain the options), the code that picks the defaults and the `--help` text to show them.  At the moment it is 2nd to write on my pile after the successor to https://github.com/diffblue/cbmc/pull/5629 .  Will you accept a "promise this will be fixed but not in this PR"?